mobile/lib/features/trade/trade_screen.dart

```diff
@@ -91,10 +101,15 @@ class TradeScreen extends StatelessWidget {
                 child: TradeTabs(
                   tabs: const [
                     "Positions",
```

Comment:
```suggestion
                    "Position",
```

Since there can ever only be 1 or 0 positions?

Reply:
Hahaha fair enough!